k8s/secrets.yml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: chatapp-secrets
+  namespace: chat-app
+type: Opaque
+data:
+  jwt: CgpleUpoYkdjaU9pSklVekkxTmlKOS5leUpTYjJ4bElqb2lRV1J0YVc0aUxDSkpjM04xWlhJaU9pSkpjM04xWlhJaUxDSlZjMlZ5Ym1GdFpTSTZJa3BoZG1GSmJsVnpaU0lzSW1WNGNDSTZNVGMxTXpNMk5UWXdNeXdpYVdGMElqb3hOelV6TXpZMU5qQXpmUS5Sa3BfblVVRUNqQlVTODRXX2FWRmNyV1VlTFJzRTh1MjNpRmhzMjBEVUZNCgo=
+  
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)